templates/vue-demo-store/components/product/ProductCard.vue

```diff
@@ -1,5 +1,8 @@
 <script setup lang="ts">
-import type { BoxLayout, DisplayMode } from "@shopware-pwa/composables-next";
+import {
```

Comment:
Let's stay with the previous imports and stick with the convention

Reply:
done